repositories.yaml

```diff
@@ -57,17 +57,15 @@ libraries:
   - library_name: sync
     description: Sync telemetry helper functionality
     notification_emails:
-      - dthorn@mozilla.com
-      - android-probes@mozilla.com
-    url: https://github.com/mozilla-mobile/firefox-android
+      - sync-team@mozilla.com
+    url: https://github.com/mozilla/application-services
     metrics_files:
-      - android-components/components/support/sync-telemetry/metrics.yaml
+      - components/sync_manager/metrics.yaml
     ping_files:
-      - android-components/components/support/sync-telemetry/pings.yaml
+      - components/sync_manager/pings.yaml
     variants:
       - v1_name: sync
-        dependency_name: org.mozilla.components:support-sync-telemetry
-
+        dependency_name: org.mozilla.appservices:syncmanager
```

Comment:
CI is failing because this name also needs to be changed on fenix and firefox_reality